docs/source/pipelines.rst

```diff
@@ -245,6 +241,56 @@ The general workflow for using MLflow Pipelines is as follows:
       local filesystem. The ``MLFLOW_PIPELINES_EXECUTION_DIRECTORY`` environment variable can
       be used to specify an alternative location for caching results.
 
+Development environments
+~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend using one of the following environment configurations to develop models with MLflow Pipelines:
+
+[**Databricks**]
+  - Edit YAML config and Python files in Databricks Repos. Open separate browser tabs for each
+    file module that you wish to modify. For example,
+    one for the pipeline config file ``pipeline.yaml``,
+    one for the profile config file ``profile/databricks.yaml``,
+    one for the driver notebook ``notebooks/databricks.py``,
+    and one for the current step (e.g. train) under development ``steps/train.py``.
+  - Use ``notebooks/databricks.py`` as the driver to run pipeline steps and inspect its output.
+  - Pin the workspace browser for easy file navigation.
+
+  .. image:: _static/images/pipelines_databricks_ui.png
+    :width: 60%
+
+[**Local with Jupyter Notebook**]
+  - Use ``notebooks/jupyter.ipynb`` as the driver to run pipeline steps and inspect its output.
+  - Edit ``pipeline.yaml``, ``steps/*.py`` and ``profiles/*.yaml`` accordingly with an IDE of your
+    choice.
+  - To run the entire pipeline, either user ``notebooks/jupyter.ipynb`` or on commandline with
+    ``mlflow pipelines run --profile local`` (change root to the project root first).
```

Comment:
```suggestion
  - To run the entire pipeline, either run ``notebooks/jupyter.ipynb`` or, on the commandline, invoke
    ``mlflow pipelines run --profile local`` (change the current working directory to the project root first).
```

Reply:
Added a few sentences in step 3 to indicate the use of profiles.